src/neuvector/values/values.yaml

```diff
@@ -10,6 +10,9 @@ manager:
   svc:
     type: ClusterIP
 
+internal:
+  autoRotateCert: true
+
```

Comment:
Wasn't able to get the scanners running without this value toggled on - seems like a good setting regardless though since it cycles the internal cert for us?

Something still seems off connectivity wise, the enforcer isn't showing up in the console so may be additional settings needed.